src/word_analysis.py

```diff
@@ -0,0 +1,12 @@
+from wordcloud import WordCloud
+import numpy as np
+from textblob import TextBlob
+import matplotlib.pyplot as plt
+
+def sentimental_analysis(text):
+    """
+    Creates an object of the TextBlob class, the sentiment attribute is used:
+    polarity: how positive or negative the sentiment is
+    """
+    sentiment = TextBlob(str(text)).sentiment
+    return sentiment.polarity, sentiment.subjectivity
```

Comment:
why do you need these ?

Reply:
I plan to add more features into the algorithm to make the outcome more accurate, this is the same reason why I made the algorithm for extracting features